superset/viz.py

```diff
@@ -1456,7 +1456,7 @@ def get_data(self, df: pd.DataFrame) -> VizData:
 
                 data.append(
                     {
-                        "key": prefix + ", ".join(series["key"]),
+                        "key": prefix + "".join(series["key"]),
```

Comment:
Good catch; I believe the key can be both `str` and `List[str]`. Due to `viz.py` not being actively maintained and being light on tests it might be a good idea to add some additional logic to guard against regressions. Perhaps we should do something like this (probably makes sense to break out `series["key"]` into a variable to avoid duplication):
```suggestion
                        "key": prefix + ", ".join(series["key"] if isinstance(series["key"], list) else [series["key"]]),
```

Reply:
Good idea, updated